icicle_v3/tests/test_field_api.cpp

```diff
@@ -204,13 +206,47 @@ TYPED_TEST(FieldApiTest, montgomeryConversion)
 #ifdef NTT_ENABLED
 TYPED_TEST(FieldApiTest, ntt)
 {
-  const int logn = 20;
+  srand(time(0));
+  const int logn = rand() % 13 + 3;
   const int N = 1 << logn;
-  auto scalars = std::make_unique<TypeParam[]>(N);
-  FieldApiTest<TypeParam>::random_samples(scalars.get(), N);
 
-  auto out_main = std::make_unique<TypeParam[]>(N);
-  auto out_ref = std::make_unique<TypeParam[]>(N);
+  // // Randomize config
+  // TODO - iterate over different configs
```

Comment:
when are you going to do it? I think it's better than random since we know what we check. Otherwise CI can pass sometimes and fail in others, based on the randomized case.

Reply:
update: since rust tests are covering many cases, this test can doesn't have to as well.

Reply:
ok